apps/server/src/server.ts

```diff
@@ -2,27 +2,30 @@ import express, { json, urlencoded } from 'express';
 import helmet from 'helmet';
 import { serve, setup } from 'swagger-ui-express';
 
-import { getServerConfig } from './config/server';
-import swaggerDoc from './config/swagger';
-import dictionaryRouter from './routes/dictionary';
-import pingRouter from './routes/ping';
+import { errorHandler, manager } from 'common';
+import { defaultAppConfig, getServerConfig } from './config/server.js';
+import swaggerDoc from './config/swagger.js';
+import pingRouter from './routes/ping.js';
 
 const serverConfig = getServerConfig();
 
+const lyricManager = manager(defaultAppConfig);
+
 // Create Express server
 const app = express();
 
+app.use(urlencoded({ extended: false }));
+app.use(json());
+
 app.use(helmet());
 
 app.use('/ping', pingRouter);
-app.use('/dictionary', dictionaryRouter);
+app.use('/dictionary', lyricManager.getRouters.dictionaryRouters);
```

Comment:
following up on [my suggestion](https://github.com/overture-stack/lyric/pull/16/files#r1509484744)

```suggestion
app.use('/dictionary', lyricProvider.routers.dictionary);
```